pandas/core/groupby/groupby.py

```diff
@@ -581,6 +581,16 @@ def wrapper(*args, **kwargs):
             if 'axis' not in kwargs_with_axis or \
                kwargs_with_axis['axis'] is None:
                 kwargs_with_axis['axis'] = self.axis
+            if name == 'hist' and kwargs.get('equal_bins', False):
+                # GH-22222
+                bins = kwargs.get('bins')
```

Comment:
Hmm wondering if we even need to use .get here - can we just access by index? I'm assuming that 'equal_bins' should always be there so a KeyError when it's not would be fine

Reply:
same py2/3 issue. In py3 `equal_bins`, `bins`, etc do not take their default values unless the code inside `hist_series` function is being run.